tests/metadata/card_metadata_integration_test.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+
+from checkout_sdk.common.enums import CardMetadataFormatType
+from checkout_sdk.metadata.metadata import CardMetadataRequest, CardMetadataBinSource
+from tests.checkout_test_utils import assert_response
+
+
+def test_should_request_card_metadata(oauth_api):
+    card_metadata_bin_source = CardMetadataBinSource()
+    card_metadata_bin_source.bin = '42424242'
+
+    card_metadata_request = CardMetadataRequest()
+    card_metadata_request.source = card_metadata_bin_source
+    card_metadata_request.format = CardMetadataFormatType.BASIC
+
+    response = oauth_api.card_metadata.request_card_metadata(card_metadata_request)
+    assert_response(response,
+                    'http_metadata',
+                    'bin',
+                    'scheme',
+                    'card_type',
+                    'card_category',
+                    'issuer_country',
+                    'issuer_country_name',
+                    'product_id',
+                    'product_type')
+    assert response.bin is not None
```

Comment:
The assert_response already evaluates that the given properties are not none, so all these asserts are redundant

Reply:
👍